libs/logging/src/main/java/org/elasticsearch/logging/LogManager.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.logging;
+
+import org.elasticsearch.logging.internal.spi.LogManagerFactory;
+
+/**
+ * A class used for creating loggers.
```

Comment:
Maybe expand this a little, e.g. 
  "A Manager of {@code Loggers}. This class consists of factory methods for creating and retrieving Loggers."